setup.py

```diff
@@ -50,7 +50,7 @@
     license='BSD 3-Clause',
     classifiers=[_f for _f in CLASSIFIERS.split('\n') if _f],
     platforms=["Windows", "Linux", "Solaris", "Mac OS-X", "Unix"],
-    install_requires=['numpy'],
+    install_requires=['numpy < 1.25'], # numpy >= 1.25 requires python 3.9
```

Comment:
Can you make this dependent on the python version?
https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#platform-specific-dependencies
So that we can use newer versions on newer python.

Reply:
Done!